README.md

```diff
@@ -26,8 +26,8 @@ If you're on a release branch (`release-<version>`) and the `<version>` portion
 We suggest that you place this action after any linting and/or testing actions to catch as many errors as possible before publishing.
 
 
-### Actions v2
-To use this in an [Actions v2](https://help.github.com/en/articles/migrating-github-actions-from-hcl-syntax-to-yaml-syntax) workflow, add the following YAML to one or more of your steps:
+### Actions
+To use this in an Actions workflow, add the following YAML to one or more of your steps:
 
 ```yaml
 - uses: primer/publish@master
```

Comment:
We've been instructing people to use `primer/publish@master`, but merging a change like this one into `master` would be a breaking change. Should we encourage people to use specific major versions, like `primer/publish@v3`? If we publish updates to `v3` we can always just move the Git tag and things will update, and if we ever introduce a v4, any current usage will be safe.

Reply:
Yeah that's a good idea!